storyscript/hub/sdk/ServiceWrapper.py

```diff
@@ -0,0 +1,132 @@
+import json
+from uuid import UUID
+
+from storyscript.hub.sdk.GraphQL import GraphQL
+
+from storyscript.hub.sdk.service.ServiceData import ServiceData
+
+
+class UUIDEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, UUID):
+            return obj.hex
+        return json.JSONEncoder.default(self, obj)
+
+
+class ServiceWrapper:
+    """
+    The ServiceWrapper provides an improved way to access storyscript hub services
+    """
+    def __init__(self, services=None):
+        self.services = {}
+        self.reload_services(services)
+
+    @classmethod
+    def from_dict(cls, dictionary=None):
+        services = []
+
+        if dictionary is not None:
+            services = dictionary
+
+        return cls(services)
+
+    @classmethod
+    def from_json(cls, jsonstr=None):
+        services = []
+
+        if jsonstr is not None:
+            services = json.loads(jsonstr)
+
+        return cls(services)
+
+    @classmethod
+    def from_json_file(cls, path=None):
+        if path is not None:
+            with open(path, 'r') as f:
+                jsonstr = f.read()
+                f.close()
+                return cls.from_json(jsonstr=jsonstr)
+
+        return cls([])
+
+    def reload_services(self, services=None):
```

Comment:
```suggestion
    def reload_services(self, services):
```

No optional parameters here either.